pyServer/GuestFishWrapper.py

```diff
@@ -268,283 +276,292 @@ def execute(self, storageUrl):
+
+                            self.WriteToResultFile(operationOutFile, "Using manifest: " + self.mode.lower() + "  [" + osType.lower() + "]" )
+                            operationNumber = 0
+                            with open(manifestFile) as operationManifest:
+                                contents = operationManifest.read().splitlines()
+                                totalOperations = len(contents)
+                                timedOut = False
+                                self.rootLogger.info("Reading manifest file from " + manifestFile + " with " + str(totalOperations) + " operation entries.")
+                                for operation in contents:
+                                    if (self.kpThread.wasTimeout == True):
+                                        if timedOut:
+                                            break;
+                                        lastGoodOperationMajorStep = operationNumber
+                                        lastGoodOperationMinorStep = 1
+                                        break
+
+                                    operationNumber = operationNumber + 1
+                                    if (operationNumber < self.modeMajorSkipTo):
+                                        strMsg = "Skipping Operation [" + str(operationNumber) + "/" + str(totalOperations) + "]: " + str(operation)
+                                        self.rootLogger.warning(strMsg)
+                                        self.WriteToResultFile(operationOutFile, strMsg)
+                                        continue
+
+                                    operation_start_time = datetime.now()
+                                    strMsg = operation_start_time.strftime('%H:%M:%S') + "  Executing Operation [" + str(operationNumber) + "/" + str(totalOperations) + "]: " + str(operation)
                                     self.WriteToResultFile(operationOutFile, strMsg)
-                                    continue
-
-                                operation_start_time = datetime.now()
-                                strMsg = operation_start_time.strftime('%H:%M:%S') + "  Executing Operation [" + str(operationNumber) + "/" + str(totalOperations) + "]: " + str(operation)
-                                self.WriteToResultFile(operationOutFile, strMsg)
-                                self.rootLogger.info(strMsg)
+                                    self.rootLogger.info(strMsg)
+
+                                    opList = operation.split(',')
+
+                                    if len(opList) < 2:
+                                        continue
+                                    
+                                    opCommand = str(opList[0]).lower().strip()
+                                    opParam1 = pathPrefix + opList[1].strip()
+                                    willScan = True
```

Comment:
Maybe add a short comment here explaining willScan flags whether the file will undergo credscan operation?

Reply:
Or to make this more symmetric with the other existing flag 'skipInspect', renaming this flag 'willScan' to 'runCredScan' to be more descriptive?

Reply:
I renamed the flag to skipCredScan.  and reversed True and False. So I didn't feel that the comment is needed. Tried to make code self-commenting, this is the best, so comments never will become out of sync with the code.